CONTRIBUTING.md

```diff
@@ -206,15 +206,15 @@ You must have the following installed on your system to contribute locally:
 
 #### Debian/Ubuntu
 
-`sudo apt install g++ make` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
+`sudo apt install build-essential` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
```

Comment:
In my tests the original instructions still worked. There wasn't any explanation about the change, however since `build-essential` is a superset of `g++ make`, this can be left as is.